tabpy-server/tabpy_server/tabpy.py

```diff
@@ -1,967 +1,10 @@
-from argparse import ArgumentParser
-import concurrent.futures
-import configparser
-from datetime import datetime
-from hashlib import md5
-from io import StringIO
-import logging
-import logging.config
-import multiprocessing
-from OpenSSL import crypto
-import os
-from pathlib import Path
-from re import compile as _compile
-import requests
-import shutil
-import simplejson
-import sys
 from tabpy_server import __version__
```

Comment:
Is this file change related to the rest of the fix?  It's really hard to tell from this one change.